Python/Product/Debugger.Concord/LocalComponent.cs

```diff
@@ -189,6 +191,44 @@ public override void Handle(DkmProcess process) {
 
                 if (PythonDLLs.GetPythonLanguageVersion(moduleInstance) != PythonLanguageVersion.None) {
                     pyrtInfo.DLLs.Python = moduleInstance;
+
```

Comment:
I'm not sure why this code is here? It looks like it's trying to do a remote attach when the python dll loads in the process. Aren't you already attached at this point? I'm not sure why we need to run the loader script here.

Also seems like this code should be moved to a separate function.